New Data Source/Resource: `azurerm_dedicated_host_group` (#5307)
terraform-providers/terraform-provider-azurerm: website/docs/r/dedicated_host_group.html.markdown

---
subcategory: "Compute"
layout: "azurerm"
page_title: "Azure Resource Manager: azurerm_dedicated_host_group"

> sidebar_current: "docs-azurerm-resource-dedicated-host-group"

Reviewer: ```suggestion
```

description: |-

>   Manage Azure DedicatedHostGroup instance.

Reviewer: ```suggestion
  Manages a Dedicated Host Group
```

---

# azurerm_dedicated_host_group

> Manage a Azure Dedicated Host Group instance.

Reviewer: ```suggestion
Manages a Dedicated Host Group
```

## Example Usage

```hcl
resource "azurerm_resource_group" "example" {
  name     = "example-rg-compute"
  location = "West Europe"
}

resource "azurerm_dedicated_host_group" "example" {
  resource_group_name         = "${azurerm_resource_group.example.name}"
  location                    = "${azurerm_resource_group.example.location}"
  name                        = "example-dedicated-host-group-compute"
  platform_fault_domain_count = 1
}
```

## Argument Reference

The following arguments are supported:

* `name` - (Required) Specifies the name of the Dedicated Host Group. Changing this forces a new resource to be created.

* `resource_group_name` - (Required) Specifies the name of the resource group the Dedicated Host Group is located in. Changing this forces a new resource to be created.

* `location` - (Required) The Azure location where the Dedicated Host Group exists. Changing this forces a new resource to be created.

* `platform_fault_domain_count` - (Required) Number of fault domains that the host group can span. Changing this forces a new resource to be created.

> * `zones` - (Optional) Availability Zone to use for this host group. Only single zone is supported. The zone can be assigned only during creation. If not provided, the group supports all zones in the region. If provided, enforces each host in the group to be in the same zone. Changing this forces a new resource to be created. 

Reviewer: ```suggestion
* `zones` - (Optional) A list of Availability Zones in which the Dedicated Host Group should be located

-> **NOTE:** At this time only a single zone is supported.
```
